### PR TITLE
Step 5 -> Add GET articles endpoint, testing and update endpoints.json

### PR DESCRIPTION
**Pull request by jonnywb**

### CODE REVIEW

**Comment by jonnywb** [comment]
Realised I forgot to add votes and comment_count to the objects - fixing now...

File: __tests__/app.test.js [hunk 1]
```diff
@@ -55,6 +55,43 @@ describe("GET /api/articles", () => {
...
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            article_img_url: expect.any(String),
```
Comment (nickkagame) [inline __tests__/app.test.js:80]: nice test.  see you've added comment count to this test in a later commit

File: models/articles.models.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.selectArticles = () => {
+  return db
+    .query(
+      "SELECT article_id, title, topic, author, created_at, article_img_url FROM articles ORDER BY created_at DESC;"
```
Comment (nickkagame) [inline models/articles.models.js]: nice query str.

**Review by nickkagame — COMMENTED** [review]
NIce work jonathan.  Pieced everything together over the several commits, and everything is there.  Crack on!

**Pull request merged**